lib/dry/system/errors.rb

```diff
@@ -58,5 +58,14 @@ def initialize(name)
         super("component identifier #{name.inspect} must be a symbol")
       end
     end
+
+    # Error raised when trying to stop a component that hasn't started yet
+    #
+    # @api public
+    ComponentNotStaredError = Class.new(StandardError) do
```

Comment:
There's a typo here. It should be `ComponentNotStaredError`.

Reply:
`ComponentNotStaredError` => `ComponentNotStartedError`

Reply:
Bloody typo 🤣